server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -619,6 +624,49 @@ export function configureWebSocketServices(
 				);
 			}
 
+			// Send signal to collaboration session from broadcast-signal endpoint
+			if (collborationSessionEventEmitter !== undefined) {
+				collborationSessionEventEmitter.on(
+					"broadcastSignal",
+					(broadcastSignal: IBroadcastSignalEventPayload) => {
+						const tenantId = broadcastSignal.tenantId;
+						const documentId = broadcastSignal.documentId;
+						const roomToBroadcastSignal: IRoom = { tenantId, documentId };
+
+						// No-op if the room (collab session) that signal came in from is different
+						// than the current room. We reuse websockets so there could be multiple rooms
+						// that we are sending the signal to, and we don't want to do that.
+						if (
+							roomToBroadcastSignal.documentId === room.documentId &&
+							roomToBroadcastSignal.tenantId === room.tenantId
+						) {
+							const contents = JSON.parse(
+								broadcastSignal.signalContent,
+							) as IRuntimeSignalMessageBody;
+							const runtimeMessage = createRuntimeMessage(contents);
+
+							socket
+								.emitToRoom(
+									getRoomId(roomToBroadcastSignal),
+									"signal",
```

Comment:
I am missing a lot of context here but this event name seems very undescriptive. Is there a reason it needs to be "signal" or can we give it a better name?

Reply:
This is how all the existing signals are sent and is the pattern for all existing code in the rest of the library. Not going to change this.